keyboards/keebio/iris/keymaps/radlinskii/README.md

```diff
@@ -0,0 +1,21 @@
+# Iris Rev. 6 keymap by radlinskii
```

Comment:
done

Reply:
Unfortunately, it is still uppercase.  This can happen depending on the OS, and/or the filesystem, if editing locally. 

You might have more success editing it in the web editor.  That or move/rename the file to a temporary name, commit and then move/rename it back, and commit again.

Reply:
Are you sure it’s still uppercase? I can see this line be all lowercase in my preview

Reply:
Or you meant the filename maybe?

Reply:
Now it's done 👍